data/saas/config/gorgias_config.yml

```diff
@@ -0,0 +1,134 @@
+saas_config:
+  fides_key: <instance_fides_key>
+  name: Gorgias SaaS Config
+  type: gorgias
+  description: A sample schema representing the Gorgias connector for Fides
+  version: 0.1.0
+
+  connector_params:
+    - name: domain
+      default_value: ethyca-test.gorgias.com
+    - name: api_key
+    - name: username
+
+  client_config:
+    protocol: https
+    host: <domain>
+    authentication:
+      strategy: basic
+      configuration:
+        username: <username>
+        password: <api_key>
+
+  test_request:
+    method: GET
+    path: /api/customers
+    query_params:
+      - name: email
+        value: <email>
+    param_values:
+      - name: email
+        identity: email
+
+  endpoints:
+    - name: customer
+      requests:
+        read:
+          method: GET
+          path: /api/customers
+          query_params:
+            - name: email
+              value: <email>
+          data_path: data
+          param_values:
+            - name: email
+              identity: email
+        update:
+          method: PUT
+          path: /api/customers/<customer_id>/
+          body: |
+            {
+               <all_object_fields>  
+            }
+          param_values:
+            - name: customer_id
+              references:
+                - dataset: <instance_fides_key>
+                  field: customer.id
+                  direction: from
+    - name: search
```

Comment:
This endpoint should be renamed to the type of data it returns, what are we returning here?